spec.md

```diff
@@ -415,18 +415,22 @@ cgroup_parent: m-executor-abcd
 
 ### command
 
-`command` overrides the default command declared by the container image (i.e. by Dockerfile's `CMD`).
+`command` overrides the default command declared by the image (i.e. by Dockerfile's `CMD`).
```

Comment:
Let's not remove the `container` image qualifier unless it's inaccurate

Reply:
Oops, good catch. Restored in c1d3e328a110d5a238bbb69cf310d67d7b3cc078